src/xml/error.cr

```diff
@@ -11,19 +11,36 @@ class XML::Error < Exception
     super(message)
   end
 
-  @@errors = [] of self
+  @@max_error_capacity = 5
+
+  @[Deprecated("This class property is deprecated. XML errors are accessible directly in the respective context via `XML::Reader#errors` and `XML::Node#errors`.")]
+  def self.max_error_capacity=(@@max_error_capacity)
+  end
```

Comment:
**Question**: if I want to disable the behavior, which is the way forward, I have to set it this value to 0, but then I get a deprecation notice :confused:

Reply:
Hm, I don't think you should actually need to disable it explicitly.
You should of course be able to do that if you want. And in that case this method should not raise a deprecation message. But we want to get rid of this method as well, so it should still be deprecated.
Maybe we should mark it as `Experimental` instead? 🤔
Or don't annotate it at all but document it that it should always be guarded by `compare_version(Crystal::VERSION, "1.13.2") > 0) && compare_version(Crystal::VERSION, "2.0.0") < 0)`.